internal/cmd/db_transfer.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/chiselstrike/iku-turso-cli/internal"
+	"github.com/chiselstrike/iku-turso-cli/internal/prompt"
+	"github.com/chiselstrike/iku-turso-cli/internal/settings"
+	"github.com/chiselstrike/iku-turso-cli/internal/turso"
```

Comment:
Should these be https://github.com/chiselstrike/turso-cli ? iku-turso-cli is old and was renamed to turso-cli.

Reply:
The code itself is working fine as is, the old name is there bc the mod wasn't renamed. Might be a good opportunity to do it now. I'll try to do it, and see if everything works fine